app/models/book.rb

```diff
@@ -264,9 +261,9 @@ def service
       Service::GOOGLE
     end
   end
-
+  
   def to_csv(options = {})
-    CSV.generate(options) do |csv|
+    CSV.generate do |csv|
```

Comment:
Not necessary to remove the options arg passed to `CSV.generate()`

Reply:
When the options arg is passed to `CSV.generate()` my test breaks with a `TypeError: no implicit conversion of Hash into String.` 

This happens regardless if my test has the options arg passed to it. I'm currently troubleshooting and will keep you posted on this

Reply:
You're right, `CSV.generate()` doesn't accept a hash. My bad.